Resources/Textures/_NF/Objects/Consumable/Food/Baked/pie.rsi/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24, mime tart slice and banana cream pie slice from rosysyntax under under CC BY-SA 4.0",
```

Comment:
Mime tart and banana cream pie? Is this copyright information accurate, or was it copied from the other pie.rsi/meta.json?

Reply:
Should be resolved.  Likely a holdover from the base pie.rsi.